src/sales/returns.md

```diff
@@ -13,6 +13,19 @@ _Returns grid_
 
 RMAs can be issued for simple, grouped, configurable, and bundle product types. However, RMAs are not available for virtual products, downloadable products, and gift cards.
 
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|Select|Select the checkbox to select the return(s) to be subject to an action, or use the selection control in the column header. Options: Select All / Deselect All / Select Visible / Unselect Visible|
+|RMA|A unique numeric identifier that is assigned to each return|
+|Requested|Date and time the return was placed|
+|Order|Unique serial number of the returned order|
```

Comment:
This isn't really a serial number per se. This is the original order number, correct?

Reply:
Yes, you are right, @dobooth 
I changed it.